cmake/BLTMacros.cmake

```diff
@@ -1363,3 +1363,42 @@ macro(blt_print_target_properties)
     unset(_is_blt_registered_target)
     unset(_is_cmake_target)
 endmacro(blt_print_target_properties)
+
+##------------------------------------------------------------------------------
+## blt_export_tpl_targets(EXPORT <export-set> NAMESPACE <namespace>)
```

Comment:
Should we have another parameter for the project specific targets?

Reply:
I don't know if installing project targets is that helpful - I think if a user is starting to do things like install project targets they already have enough CMake knowledge to do this. The main advantage to this macro is to hide the boilerplate, and also hide the internal target names (because we cannot use the aliases).